packages/venia-concept/src/components/Head/title.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Title as _Title } from 'react-head';
+
+export default props => (
+    <_Title {...props} data-synthetictag>
```

Comment:
Oh, so we're adding the synthetic tag? 

Why does `HeadProvider` need to filter them out then? How could they exist before `HeadProvider` is mounted?

Reply:
Explained in the above comment.